docs/release-notes/k8s-context-size-limit.rst

```diff
@@ -2,5 +2,5 @@
 
 **Improvements**
 
--  Kubernetes: Increased the file context limit for Notebooks, Commands, Tensorboards, and Shells
+-  Kubernetes: Increase the file context limit for notebooks, commands, TensorBoards, and shells
```

Comment:
This is a release note file that was included in the current release and is about to be deleted, so you should not change it. The text was updated in the release branch.